cmd/collect.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"github.com/numaproj-labs/numaflow-perfman/collect"
+	"github.com/numaproj-labs/numaflow-perfman/util"
+)
+
+var (
+	Name    string
+	Last    int
+	Metrics []string
+)
+
+var validMetrics []string
+var metricsMatrix = map[string][]collect.MetricObject{
```

Comment:
Can we remove `validMetrics` and directly use `metricsMatrix`?

Reply:
The cobra flag `Metrics` is of type `StringSliceVarP` which requires the provided default value to be `[]string`. Is there another way/or flag we can use?

Reply:
I see, so we need to extract the key set from the map. Sure, we can keep as it is.